System/Random.hs

```diff
@@ -190,9 +166,6 @@ module System.Random
 
   -- * References
   -- $references
-
-  -- * Internals
-  , bitmaskWithRejection -- FIXME Export this in a better way, e.g. in System.Random.Impl or something like that
```

Comment:
@curiousleo I don't think we should export bitmask with rejection function, definitely not until and if we get some internal module in place, cause all of those functions expect invariant on their inputs that are not enforced, therefore they shouldn't be a part of public API. For that reason I also disabled tests of `bitmaskWithRejection` ( I also removed the function itself since only monadic versions are being used)

Reply:
Makes sense.